test-plan/eventing/control-plane/event-delivery/01-00-channel-ack.yaml

```diff
@@ -0,0 +1,70 @@
+# Receives and logs messages.
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sink-ack-conformance
+  namespace: conformance
+  labels:
+    component: conformance-sink
+    app: sink-ack-conformance
+spec:
+  containers:
+  - name: sink-ack-conformance
+    image: pmercado/recordevents
+    env:
+    - name: POD_NAME
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.name
+    - name: SYSTEM_NAMESPACE
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.namespace
+    - name: EVENT_GENERATORS
+      value: receiver
+    - name: EVENT_LOGS
+      value: logger
```

Comment:
IIRC for automated tests the `recorder` log is used. It generates structured k8s events that are easier to parse.

Reply:
Yes agree, but while drafting this I had to delete `Events` manually when executing multiple times the same test.
I'll give it a second shot.

Reply:
Recorder seems to use the Pod name to generate the event name. Re-creating a Pod with the same name leads to errors at tests.

One solution is to use `generatedName` which must be what automated tests do, but then I'll need to re-setup some labels so that each test case can be cleaned up by doing something like:
```
kubectl delete all -n conformance -l test-case=broker-delivery-successful
```

I'll work on that.

Reply:
fixed https://github.com/knative/specs/pull/82/commits/e5c6215f8136df1e42299bd03c01acc589ce4537